frontend-react/src/config/endpoints/messageTracker.ts

```diff
@@ -41,6 +41,7 @@ export interface RSMessageDetail {
 }
 
 export interface MessageListResource {
+    id?: number;
```

Comment:
nit, would this ever be null?

Reply:
This piece of code got around typing that: 

https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/components/Table/ColumnData.tsx#L80-L82

Reply:
Going to update the MessageTrackerMockServer:

https://github.com/CDCgov/prime-reportstream/blob/fe8425f4b8897181204d47402133843c5738d688/frontend-react/src/__mocks__/MessageTrackerMockServer.ts#L28-L47

Let me know if you see any problems @penny-lischer